docs/conf.py

```diff
@@ -108,7 +108,7 @@
 #
 # html_sidebars = {}
 
-html_logo = '_static/images/firesim_logo_small.png'
+html_logo = '_static/firesim_logo_small.png'
```

Comment:
how did this work before?

Reply:
The CI requires warnings to be errors... so IIRC this was a warning that was not addressed.